jdaviz/configs/default/plugins/model_fitting/model_fitting.vue

```diff
@@ -289,6 +289,12 @@
             </span>
           </v-row>
 
+          <v-row v-if="non_finite_uncertainty_mismatch">
+            <span class="v-messages v-messages__message text--secondary" style="color: red !important">
+                Finite values in the selected data will be omitted in the fit due to corresponding non-finite uncertanties.
+            </span>
+          </v-row>
+
```

Comment:
I'm not seeing this in the plugin. How can I make it appear?

Reply:
```
uncertainty=StdDevUncertainty([1, 1, np.nan, np.nan, np.nan, np.nan]*u.Jy) 
spec = Spectrum1D(flux=[1, 2, 3, 4, 5, 6]*u.Jy, uncertainty=uncertainty)
specviz = Specviz()
specviz.load_data(spec)
specviz.show()
```

that should trigger it. it will appear just above the 'fit model' button.

Reply:
yep, if the whole array is none then the first snackbar message appears. if its not and there are some non finite uncertainties, then the message in the plugin is displayed